slam-libraries/viam-orb-slam3/orbslam_server_v1_test.cc

```diff
@@ -110,7 +110,7 @@ BOOST_AUTO_TEST_CASE(ParseAndValidateArguments_valid_config_no_map_rate_sec) {
                               "-data_rate_ms=200",  "-map_rate_sec="};
     SLAMServiceImpl slamService;
     utils::ParseAndValidateArguments(args, slamService);
-    BOOST_TEST(slamService.map_rate_sec.count() == chrono::seconds(0).count());
+    BOOST_TEST(slamService.map_rate_sec.count() == chrono::seconds(60).count());
```

Comment:
[q] Why does this have to change? I would have assumed that it would have had to be equal to a value of `60` before, too?

Reply:
in this test no value is given and so it was defaulted to 0. this behavior is not desired now that localization is 0. We can either have it default to extremely high map rate value to avoid saving a map OR treat it like the data rate and throw an error.

Reply:
I'm in favor of option 2

Reply:
hmn let me think this through: If no `map_rate_sec` is provided via RDK, that means something went wrong, correct? Since RDK always should be sending a value for `map_rate_sec`. If we run orbslam locally, though, and the user doesn't provide a value for `map_rate_sec`, I would assume that the behavior should be the same as if the user doesn't provide a value through the config - so setting it to a default value of `60` would make sense to me, since it is coherent with how we handle it through RDK, and since providing this parameter should be optional. Am I making sense?

Reply:
Discussed this in person - agreed to throw an error